assessments/MAP_Fluency/templates/assessments.jsont

```diff
@@ -0,0 +1,50 @@
+{
+  "assessmentIdentifier": "{{assessmentIdentifier}}",
+  "assessmentTitle": "{{assessmentTitle}}",
+  "assessmentFamily": "{{assessmentFamily}}",
+  "namespace": "{{namespace}}",
+  "assessedGradeLevels": [
```

Comment:
@ryanaguilar could you move these grade level descriptors to a seed, as is suggested here (with example linked too https://edanalytics.slite.com/app/docs/ovhlvaSRkIfFOh#75127bdd)

I think the idea is that by making them in a seed, implementations can override with local descriptors easily, which is pretty common

Reply:
(and same for any other cases of ed-fi default descriptors, e.g. subjects)